src/fidesops/ops/schemas/manual_webhook_schemas.py

```diff
@@ -41,6 +47,33 @@ def check_for_duplicates(
         """
         Verify that pii_fields and dsr_package_labels are unique.
 
+        Set the dsr_package_label to a snake_cased lower case version of pii field if it doesn't exist.
+        """
+        unique_pii_fields: Set[str] = {field.pii_field for field in value}
+        if len(value) != len(unique_pii_fields):
+            raise ValueError("pii_fields must be unique")
```

Comment:
Are we translating these `ValueError`s into http exceptions in `patch_access_manual_webhook` and `create_access_manual_webhook` endpoints? I think `HTTP_400_BAD_REQUEST` makes sense for these

Reply:
Yes, http exception conversion is already automatically handled by FastAPI/Pydantic - any ValueErrors we raise in these validators end up getting converted to 422's https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions.  

While I could add a custom exception handler, I think this is more consistent with our other endpoints, in that most of the time if something fails at the pydantic schema level, we're raising this default 422.  

There's also a unit test included with this work that confirms an http exception is thrown!

Reply:
I'd forgotten about that automatic conversion, this makes sense!